rebar.lock

```diff
@@ -78,23 +74,24 @@
   {git,"git://github.com/comtihon/erlang-pbkdf2.git",
        {ref,"7076584f5377e98600a7e2cb81980b2992fb2f71"}},
   1},
- {<<"plumtree">>,
-  {git,"git://github.com/erlio/plumtree.git",
-       {ref,"34cb2ab0b194992e49558ebf7ed4c9e7ff7e0c8a"}},
-  0},
  {<<"poolboy">>,{pkg,<<"poolboy">>,<<"1.5.1">>},0},
  {<<"ranch">>,{pkg,<<"ranch">>,<<"1.2.1">>},0},
  {<<"recon">>,{pkg,<<"recon">>,<<"2.3.2">>},0},
  {<<"riak_dt">>,
   {git,"git://github.com/basho/riak_dt.git",
        {ref,"f7981d4ad7407ddc085f133f204dd71bf9d50c56"}},
-  1},
+  0},
  {<<"riak_sysmon">>,{pkg,<<"riak_sysmon">>,<<"2.1.2">>},0},
+ {<<"rocksdb">>,{pkg,<<"rocksdb">>,<<"0.20.0">>},0},
  {<<"sext">>,
   {git,"git://github.com/uwiger/sext.git",
        {ref,"35520daea8ddc48569249ad19637f4cc75fbd03d"}},
   0},
  {<<"ssl_verify_fun">>,{pkg,<<"ssl_verify_fun">>,<<"1.1.1">>},1},
+ {<<"swc">>,
+  {git,"git://github.com/dergraf/ServerWideClocks.git",
```

Comment:
shouldn't this be the `vernemq` org repo? Or is it just to test sth out?

Reply:
It should.. but my PR isn't merged yet, I thinnk